Library/Homebrew/cmd/update.rb

```diff
@@ -41,8 +41,10 @@ def update
     master_updater.pull!
     master_updated = master_updater.updated?
     if master_updated
-      puts "Updated Homebrew from #{master_updater.initial_revision[0, 8]} " \
-           "to #{master_updater.current_revision[0, 8]}."
+      initial_revision_short = shortenRevision(master_updater.initial_revision)
+      current_revision_short = shortenRevision(master_updater.current_revision)
+      puts "Updated Homebrew from #{initial_revision_short} " \
+           "to #{current_revision_short}."
```

Comment:
You guessed well and not everything is set in stone, so other maintainers might disagree with me. (But I think it's okay to keep things short in this case for the reasons explained above.)